demos/sslecho/echecho.c

```diff
@@ -100,7 +100,7 @@ static int configure_ech(SSL_CTX *ctx, int server,
     OSSL_ECHSTORE *es = NULL;
     BIO *es_in = BIO_new_mem_buf(buf, len);
 
-    if (es_in == NULL || (es=OSSL_ECHSTORE_init(NULL,NULL)) == NULL)
+    if (es_in == NULL || (es = OSSL_ECHSTORE_init(NULL,NULL)) == NULL)
```

Comment:
ouch, I did not notice before, please also add a space after `,`

Reply:
heh - me neither, but done now